source/install/config-ssl-http2-nginx.rst

```diff
@@ -14,95 +14,140 @@ You can use any certificate that you want, but these instructions show you how t
 .. note::
    If Let’s Encrypt is enabled, forward port 80 through a firewall, with `Forward80To443 <https://docs.mattermost.com/administration/config-settings.html#forward-port-80-to-443>`__ ``config.json`` setting set to ``true`` to complete the Let’s Encrypt certification.
 
-**To configure NGINX as a proxy with SSL and HTTP/2:**
+**To configure NGINX as a proxy with SSL and HTTP/2**
+
+If you're looking for additional Let's Encrypt/Certbot assistance you can access their documentation `here <https://certbot.eff.org>`_ .
 
 1. Log in to the server that hosts NGINX and open a terminal window.
-2. Install git.
 
-  If you are using Ubuntu or Debian:
+2. Open the your Mattermost ``nginx.conf`` file as *root* in a text editor and update the IP address in the ``upstream backend`` to point towards Mattermost, and the ``server_name`` to be your domain for Mattermost.
 
-  ``sudo apt-get install git``
+.. note::
+   On Ubuntu this file is located at ``/etc/nginx/sites-available/``. If you don't have this file run ``sudo touch /etc/nginx/sites-available/mattermost``.
+   On CentOS/RHEL this file is located at ``/etc/nginx/conf.d/``. If you don't have this file run ``sudo touch /etc/nginx/conf.d/mattermost``.
+   
+.. code-block:: none
 
-  If you are using RHEL:
+   upstream backend {
+       server {ip}:8065;
```

Comment:
Maybe we should explicitly ask to replace `{ip}` with the upstream IP?

Reply:
I can change the verbiage to that if you think it'll make more sense. They just need to plug in whatever their upstream IP is. Whether that's localhost, 10.0.0.x, etc.

Reply:
I call out on line 23 to update this upstream IP to something that points to Mattermost. I'll update a comment above it to make it more explicit